src/librustc_typeck/diagnostics.rs

```diff
@@ -1972,6 +1972,88 @@ To learn more about traits, take a look at the Book:
 https://doc.rust-lang.org/book/traits.html
 "##,
 
+E0174: r##"
+This error occurs because of the explicit use of unboxed closure methods
+that is an experimental feature in current Rust version.
+
+Example of erroneous code:
+
+```compile_fail
+ fn foo<F: Fn(&str)>(mut f: F) {
```

Comment:
I just noticed: why this functions' lines have an extra blank space at the beggining?